### PR TITLE
Enable issue template for better creation of issues

### PR DESCRIPTION
**Pull request by sunishsheth2009**
## What changes are proposed in this pull request?
Enable issue template for better creation of issues

## How is this patch tested?
N/A

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components
- [ ] `area/regression`: Sklearn regression pipeline example
- [x] `area/build`: Build and test infrastructure for MLflow pipeline example

<a name="release-note-category"></a>
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Review by jinzhang21 — COMMENTED** [review]
Could you add examples how these templates look like?

**Comment by sunishsheth2009** [comment]
> Could you add examples how these templates look like?

It should be similar to one from mlflow: https://github.com/mlflow/mlflow/issues/new/choose
Just removed some sections that didn't seem relevant.

**Review by jinzhang21 — COMMENTED** [review]
I'm a bit hesitant to duplicate the MLflow issue guideline here. Can you make changes to guide users to report to the main MLflow repo about execution issues? It's easier to track instead of asking the OSS team to look MLP templates for issue tracking as well. Imagine we have 100 templates published next year. Who's going to check all those template repos for issue report?

**Comment by dbczumar** [comment]
> I'm a bit hesitant to duplicate the MLflow issue guideline here. Can you make changes to guide users to report to the main MLflow repo about execution issues? It's easier to track instead of asking the OSS team to look MLP templates for issue tracking as well. Imagine we have 100 templates published next year. Who's going to check all those template repos for issue report?

+1

File: .github/ISSUE_TEMPLATE/installation_issue_template.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Installation Issues
+description: Use this template for reporting bugs encountered while installing MLflow.
```
Comment (jinzhang21) [inline .github/ISSUE_TEMPLATE/installation_issue_template.yaml]: installing MLflow -> installing this MLflow pipeline template.

File: .github/ISSUE_TEMPLATE/installation_issue_template.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: Installation Issues
```
Comment (jinzhang21) [inline .github/ISSUE_TEMPLATE/installation_issue_template.yaml]: I don't think we have to single installation out. It's a relatively simple template repo. Issues can be aggregated under the generic "BUG" label. Easier to triage there.

**Review by jinzhang21 — APPROVED** [review]
Fantasitic! Thank you, Sunish!

**Pull request merged**